dnf5/context.cpp

```diff
@@ -202,7 +202,7 @@ class RpmTransCB : public libdnf::rpm::TransactionCallbacks {
             case libdnf::transaction::TransactionItemAction::REMOVE:
             case libdnf::transaction::TransactionItemAction::REPLACED:
                 break;
-            case libdnf::transaction::TransactionItemAction::REASON_CHANGE:
+            default:
```

Comment:
In DNF5 we try to avoid default cases, because it hides missing handling in switches during compilation and it is discovered during runtime when value is used. If there is no default case and we add a new value to enum, the compilation will tell us all places where enum is used in a switch - safer approach. Is there any reason why you would prefer to use default case here?

Reply:
I see, that makes sense. I listed all the options, then.